packages/app/src/runs/utils/day.js

```diff
@@ -1,8 +1,12 @@
 import dayjs from 'dayjs'
 import relativeTime from 'dayjs/plugin/relativeTime'
 import duration from 'dayjs/plugin/duration'
+import customParseFormat from 'dayjs/plugin/customParseFormat'
+import utc from 'dayjs/plugin/utc'
 
 dayjs.extend(relativeTime)
 dayjs.extend(duration)
+dayjs.extend(customParseFormat)
+dayjs.extend(utc)
```

Comment:
Ah, my earlier comment regarded this file. Could we remove the other `extend` calls to `dayjs` in favor of using this instance?